Makefile

```diff
@@ -84,9 +84,7 @@ SHELL = /usr/bin/env bash -o pipefail
 E2E_TEST_FLAGS = "--skip-deletion=false" -timeout 15m # See README.md, default go test timeout 10m
 
 # Default image-build is to not use local odh-manifests folder
-# set to "true" to use local instead
-# see target "image-build"
-IMAGE_BUILD_FLAGS = --build-arg USE_LOCAL=false
+IMAGE_BUILD_FLAGS ?= --build-arg FETCH_MANIFESTS=true
```

Comment:
It does not make sense for the usage like in the README.md. Command line variables have priority in make anyway. Even over environment regardless of -e flag.

But may be having a separate variable to avoid passing `--build-arg` can be a good idea, what do you think?

Reply:
> > But may be having a separate variable to avoid passing --build-arg can be a good idea, what do you think?
> 
> I am on the verge here. I have
> 
> ```shell
> alias image='make image -e IMAGE_BUILD_FLAGS="--build-arg FETCH_MANIFESTS=false" -e IMG=quay.io/maistra-dev/opendatahub-operator:$\(VERSION\) -e IMAGE_BUILDER=docker -e VERSION=dev-$(git branch --show-current)'
> ```
> 
> so I'm good.

Great! Just keep in mind that `-e` is redundant here, it works differently for make than to podman-run.

> 
> Would it make it easier for you to have such a flag?

No actually, I haven't ever used that in practice :). So I personally would vote to avoid the change at all. Especially because it's unrelated to the purpose of the patch.

Reply:
@bartoszmajsak was thinking about avoiding go version setting in 4 places. One of the ideas was getting it from go.mod and instead of patching Dockerfile, pass it as the argument (similar to toolbox), but then your alias will be broken.